src/components/Gauge.js

```diff
@@ -76,119 +53,141 @@ const StatusGaugeWrapper = styled.div`
   }
 `
 
-const GaugeTotalText = styled.text`
-  font-family: ${font};
-  font-size: ${props => px(props.count > 999 ? 20 : 26)};
-  fill: #5e5e5e;
-  letter-spacing: -1px;
-  text-anchor: middle;
-  transform: translateX(0);
-`
+let DoughnutChart = null
```

Comment:
!!! The Gauge will be appearing several times in the page, so this should probably be a property of the StatusGauge class.